snare/tests/test_cloner_add_scheme.py

```diff
@@ -16,22 +18,28 @@ def setUp(self):
         self.expected_err_url = yarl.URL("http://example.com/status_404")
         self.max_depth = sys.maxsize
         self.css_validate = False
-        self.handler = Cloner(self.url, self.max_depth, self.css_validate)
+        self.handler = BaseCloner(self.url, self.max_depth, self.css_validate)
 
     def test_trailing_slash(self):
         self.url = "http://example.com/"
+        if not self.handler:
```

Comment:
you have this check in multiple tests, why not checking it in setup?

Reply:
I had kept it for linter checks, but with my recent push I have now shifted it to `setUp` as you suggested.